src/components/Soilage.js

```diff
@@ -0,0 +1,21 @@
+import { Component } from 'geotic'
+import { colors } from '../lib/graphics'
+
+export default class Soilage extends Component {
+    static allowMultiple = true
+
+    static properties = {
+        color: colors.defaultColor,
+        name: 'blood',
+        sourceName: '',
+        sourceEntityId: ''
```

Comment:
if you're looking to reference an entity, you can use the special `<Entity>` value!

```js
static properties = {       
    ...
    sourceEntity: '<Entity>'
}
```

it will `serialize` and `deserialize` properly when the entity is destroyed, the value will be set to `undefined` as well

```js
entityA.soilage.sourceEntity = ecs.createEntity();

entityA.soilage.sourceEntity.id;
```

There is also `<EntityArray>`

Reply:
![giphy-12](https://user-images.githubusercontent.com/925980/83343255-67126980-a2bd-11ea-9e58-261b04cb2786.gif)

Nice! Gonna have to give it a try :)